frontend/cypress/e2e/change-complaint-status.cy.ts

```diff
@@ -70,4 +71,62 @@ describe("Complaint Assign and Status Popover spec", { scrollBehavior: false },
       //  .should("have.length.at.least", 1);
     });
   });
+
+  it("User can not change the status when Review is not complete ", () => {
```

Comment:
@dmitri-korin-bcps, Sorry there was a change in requirements by @Scarlett-Truong  while you were working on this.   There is no longer the ability to change status from the list view.   For now you can just update these tests to be it.skip so that we will have the tests for when we re-enable the feature.